.env.example

```diff
@@ -1,4 +1,2 @@
-REACT_APP_API_URL=http://your-api-url
-REACT_APP_CHAIN_TYPE=testnet
 REACT_APP_MAINNET_URL=http://localhost:3000
 REACT_APP_TESTNET_NAME=testnet
```

Comment:
These 2 variables are not really used.

`MAINNET_URL` is actually defined by `config.BASE_URL`(https://github.com/nervosnetwork/ckb-explorer-frontend/blob/develop/src/constants/common.ts#L112-L113) which is from `REACT_BASE_URL` and default to be `explorer.nervos.org` in https://github.com/nervosnetwork/ckb-explorer-frontend/blob/develop/src/config.ts#L4

And `TESTNET_NAME` is also defined in code https://github.com/nervosnetwork/ckb-explorer-frontend/blob/develop/src/constants/common.ts#L107-L110

Reply:
OK, then I remove this file directly, pushed